src/components/Tabs/Tabs.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { writable } from "svelte/store";
+    import TabsHead from "./TabsHead.svelte";
+    import TabItem from "./TabItem.svelte";
+
+    type Item = {
+        label: string;
+        href: string;
+        onClick?: () => void;
+    };
+
+    export let defaultActive: string = "0";
```

Comment:
@tsejerome why string, not int? Is it the index of the active tab?

Reply:
Yes it is the index of the active tab. I also agree that it is weird to be a string too.

I think that it shd be
1) renamed as `defaultActiveTabIndex`
2) type of `number` 

I tried to do the development with minimal changes so I didn't change the original code copied from holdex
If it is okay to have more modifications then I will rename and refactor this part